script/remove_offline_runners.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2022 Canonical
+# See LICENSE file for licensing details.
+
+import sys
+import logging
+
+import requests
+
+ORG = "<your org>"
+TOKEN = "<your token>"
```

Comment:
I guess this was intended to be a quick script to be run as simple as possible, I think we could turn this in to arguments with the argparse library later :) Out of the scope of the PR, please feel free to close this comment